### ISSUE TITLE
Improve Javascript/Node markdown syntax highlighting

### ISSUE THREAD

**Issue by CrabDude** [codepath/nodejs_guides#19]
The [Github](https://github.com/codepath/nodejs_guides/wiki/Control-flow) syntax highlighting is much better than the [CodePath](http://guides.codepath.com/nodejs/Control-flow) syntax highlighting.

**Comment by nesquena** [comment]
Yes, it's a valid point. It's especially bad on node syntax highlighting for some reason. Will have to switch back to pygments eventually.
